app/javascript/elm/src/Main.elm

```diff
@@ -142,6 +145,7 @@ init flags url key =
         , logoNav = flags.logoNav
         , linkIcon = flags.linkIcon
         , resetIcon = flags.resetIcon
+        , tooltipIcon = flags.tooltipIcon
```

Comment:
One thing we could do is

```
type Path = Path String

type alias Model =
  { ...
  , tooltipIcon : Path
  }
```

And here `tooltipIcon = Path flags.tooltipIcon`. That way we can't pass a string when the function needs a `Path`. Then you would need a function like

```
path : Path -> String
path (Path str) = str
```

Reply:
good idea, and it will be easily distinguishable from other strings among function attribute types. I didn't like adding another String cos it looked random and it isn't. thx for showing it, I'll refactor but don't know if right away

Reply:
@3v0k4 https://github.com/HabitatMap/AirCasting/pull/202/commits/bd12153191ed06fd13118e76b12b0c80da7ac5fb let me know if you have any comments